docs/source/glossary.rst

```diff
@@ -5,17 +5,74 @@ Glossary
 
 .. glossary::
 
+   precomputed
+      A chunked storage format for volumetric data that relies on ``infofiles`` to keep track of the volume's metadata, used by neuroglancer. See the `documentation <https://github.com/google/neuroglancer/blob/master/src/neuroglancer/datasource/precomputed/README.md>`_.
+
    CloudVolume, cv
-      TODO
+      A serverless Python client for reading and writing in precomputed format. Supports local, AWS S3, and GCS. Relies on service tokens. See the `documentation <https://github.com/google/neuroglancer/blob/master/src/neuroglancer/datasource/precomputed/README.md>`_.
 
    Neuroglancer, ng
-      TODO
+      WebGL-based viewer for volumetric data in precomputed format. See the `documentation <https://github.com/google/neuroglancer>`_.
 
    GCP
-      TODO
+      Google Cloud Platform. Supported by ``zetta_utils.mazepa`` for executing tasks remotely using Kubernetes.
 
    SQS
-      TODO
+      Amazon's Simple Queue Service. Supported by ``zetta_utils.mazepa`` for keeping track of task executions when executing tasks remotely.
 
    MIP, MIP level
-      TODO
+      Multum in parvo. Refers to the resolution of volumetric data, with MIP 0 being the highest resolution and successive MIPs being downsampled by a factor of 2 in XY. Used to reduce the data size.
+
+   EM
+      Electron microscopy.
+
+   SEM
+      Scanning Electron Microscopy.
+
+   FIB-SEM
+      Focused Ion Beam Scanning Electron Microscopy.
+
+   TEM
+      Tunneling Electron Microscopy.
```

Comment:
Should be Transmission Electron Microscope

Reply:
Oops, sleep deprivation is real....